### ISSUE TITLE
🐛 Settings provided in `config.lua` aren't applied via Docker deployment

### ISSUE THREAD

**Issue by dr460nf1r3** [neon-mmd/websurfx#305]
### Description

I recently set up an instance of websurfx via the official Docker image and Docker Compose.
The following config is used:
~~~
logging = false
debug = false
threads = 8

port = "8080"
binding_ip = "0.0.0.0"
production_use = true
request_timeout = 10
rate_limiter = {
	number_of_requests = 20,
	time_limit = 3,
}

safe_search = 2
colorscheme = "dracula"
theme = "simple"

redis_url = "redis://websurfx_redis:6379"

upstream_search_engines = {
	DuckDuckGo = true,
	Searx = true,
}
~~~
However, this does not apply to the instance. The config is mounted into the container like that:
~~~
  # Websurfx search engine
  websurfx:
    image: neonmmd/websurfx:latest
    container_name: websurfx
    ports: [8086:8080]
    depends_on: [websurfx_redis]
    links: [websurfx_redis]
    volumes: [./websurfx/:/etc/xdg/websurfx/]
  websurfx_redis:
    container_name: websurfx_redis
    image: redis:latest
~~~

### Screenshots

![image](https://github.com/neon-mmd/websurfx/assets/12834713/3358d180-451a-48f9-8300-2641dc75a497)

### Do you want to work on this issue?

No

**Comment by neon-mmd** [comment]
Sorry for the late reply.

Thanks :heart: for reporting this issue. Actually, this issue is known and is already being marked to be worked on in the next release cycle (for v2.0.0). So we will be working on this issue soon :slightly_smiling_face: .

**Comment by dr460nf1r3** [comment]
Huh?

**Comment by neon-mmd** [comment]
> Huh?

The stale bot got kicked in because the issue didn't see much activity for a long time, but don't worry, We will not let this happen again. Also, I think since this issue is planned to be worked on soon. So, I think it would be better to put this issue on `awaiting triage`. What do you think? :slightly_smiling_face:

**Comment by dr460nf1r3** [comment]
I see, sounds good to me! :)

**Comment by neon-mmd** [comment]
Thanks :heart: again for taking the time to report the issue, and sorry for the long delay to fix this issue. Actually, we had to get some issue done before working on this issue. But now this issue has been fixed with the PR #462 in the rolling release of the project and would be added soon to the next stable release `v2.0.0` which plan to get it released by next year on october 3rd. As such we are closing this issue, but if you still face any issue feel free to reopen this issue. Also, if you want to use the patched version of the project, then we would suggest using the rolling version of the project, which also tends to have new features and many speed improvements than the stable release. :slightly_smiling_face: 

Additionally, we have noticed that you have hosted your own instance of `websurfx` so we would suggest adding it [here](https://github.com/neon-mmd/websurfx/blob/rolling/docs/instances.md) by opening a PR on our project. :slightly_smiling_face:
